### PR TITLE
Update plugin to publish in SonarQube's Marketplace

### PR DESCRIPTION
**Pull request by louisjdmartin**
This PR fix issues from: https://community.sonarsource.com/t/first-release-cnesreport-3-0-1/13368/3

# Fixed issues
* Fix: Files are deleted from temp directory after usage
* Fix: NullPointerException in LanguageProvider
* Fix: On CSV export, some export can fails if SonarQube API return list of list of String instead of list of Strings

# Proposed changes
* Display an error when project is not provided and tell user to provide it with -t argument
* Add more information about plugin in README file
* New tests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -113,7 +113,7 @@ java -Dhttps.proxyHost=https://myproxy -Dhttps.proxyPort=42
   <td><b>2.0.0<br/>Standalone only</b></td>
   <td><b>2.1.0<br/>Standalone only</b></b></td>
   <td><b>2.2.0<br/>Standalone + Plugin</b></td>
-  <td><b>3.0.x<br/>Standalone + Plugin</b></b></td>
+  <td><b>3.x.x<br/>Standalone + Plugin</b></b></td>
  </tr>
  <tr>
   <td><b>3.7.x (LTS)</b></td>
```
Comment (begarco) [inline README.md]: 3.7.x ????

Comment (louisjdmartin) [reply README.md]: It's the Sonarqube version, not the cnes-report version

File: src/main/java/fr/cnes/sonar/report/providers/LanguageProvider.java [hunk 2]
```diff
@@ -64,7 +64,14 @@ public String getLanguage(final String languageKey)
         if(languages.isEmpty()){
             this.getLanguages();
         }
-        return this.languages.get(languageKey).getName();
+        String languageName;
```
Comment (begarco) [inline src/main/java/fr/cnes/sonar/report/providers/LanguageProvider.java]: You can initialize this variable with "?" and delete the else clause.

File: src/test/ut/java/fr/cnes/sonar/report/ReportTest.java [hunk 3]
```diff
@@ -57,4 +62,35 @@ public void defaultReportValuesTest() {
         assert(report.getMeasures().isEmpty());
     }
 
+    @Test (expected = IllegalStateException.class)
+    public void emptyExecuteTest() throws Exception{
+        ReportCommandLine.execute(new String[0]);
+    }
+
+    String TARGET = "./target/zip";
```
Comment (begarco) [inline src/test/ut/java/fr/cnes/sonar/report/ReportTest.java]: Put this line with other members declaration, and set it private and static.

File: src/main/java/fr/cnes/sonar/plugin/tools/ZipFolder.java [hunk 4]
```diff
@@ -58,8 +59,9 @@ public static void pack(String sourceDirPath, String zipFilePath) throws IOExcep
                     });
         }
         finally {
```
Comment (begarco) [inline src/main/java/fr/cnes/sonar/plugin/tools/ZipFolder.java]: Fix this try-with-resources misuse.

**Pull request merged**